frontend/pages/upload/bmm.vue

```diff
@@ -81,7 +81,7 @@ const reset = () => {
                 <div v-if="metadataIsSet"
                     class="flex flex-col gap-4 p-4 transition"
                 >
-                    <h1 class="text-xl font-bold">Upload files for "{{metadata.title[0]}}"</h1>
+                    <h1 class="text-xl font-bold">Upload files for "{{form.track.title}}"</h1>
```

Comment:
Is `form.track` always defined here? The `form` variable seems to only be initialised with `title` and `environment` properties

Reply:
The whole thing is in and `if metadataIsSet` https://github.com/bcc-code/bcc-media-tools/pull/25/files#diff-1b79de5af0a3d80171e6bef0db0b2bbe959b295377e3e9b6cfaec17240197f8bR81
So as far as I understand it is always set.